### PR TITLE
BaSP-MR-38: Admins get and post unit test

### PR DESCRIPTION
**Pull request by GleViDe**

### CODE REVIEW

**Review by TomasRomanoo — APPROVED** [review]
Good job! its approved for me

**Review by FrancoLelli — APPROVED** [review]
Nice Job!!

File: package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
     "description": "app_name description",
     "main": "index.js",
     "scripts": {
-        "test": "jest",
+        "test": "jest --coverage",
```
Comment (julianv97) [inline package.json:7]: Go back with this change

**Review by julianv97 — CHANGES_REQUESTED** [review]

**Review by lorenzomauro59 — APPROVED** [review]
Approved! nice work!

File: package.json (hunk 1 above)
Comment (GleViDe) [reply package.json:7]: On it!!

**Review by julianv97 — APPROVED** [review]

**Pull request merged**